spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitConnectionFactoryBean.java

```diff
@@ -565,6 +568,27 @@ public void setMetricsCollector(MetricsCollector metricsCollector) {
 		this.connectionFactory.setMetricsCollector(metricsCollector);
 	}
 
+	/**
+	 * Set to true to enable amqp-client automatic recovery. Note: Spring AMQP
+	 * implements its own connection recovery and this is generally not needed.
```

Comment:
Yes; the monitor task detects closed channels and schedules the consumer to be restarted.